tasmota/tasmota_xnrg_energy/xnrg_19_cse7761.ino

```diff
@@ -635,6 +650,8 @@ void Cse7761DrvInit(void) {
     Energy->use_overtemp = true;                 // Use global temperature for overtemp detection
     TasmotaGlobal.energy_driver = XNRG_19;
   }
+  Settings->flag2.frequency_resolution = 2;      // 0.01Hz resolution
```

Comment:
Pls do not do this. Although I understand why you do this, some users may not like it. They can always use the correct commands to change these parameters.

Reply:
I've thinking about some option like: